### PR TITLE
ELBv2: added validation for target group

### PR DESCRIPTION
**Pull request by macnev2013**
# Motivation
- This PR aims to make target group in parity with AWS.

# Added
- Parameter validation for `create_target_group`
- Default values for `create_target_group`
- Adjusted test cases accordingly
- Fixed `create_target_group` response

# Notes
- Related PR for AWS Validated Test Case: https://github.com/localstack/localstack-ext/pull/2064

### CODE REVIEW

File: moto/elbv2/models.py [hunk 1]
```diff
@@ -1080,6 +1087,90 @@ def create_target_group(self, name: str, **kwargs: Any) -> FakeTargetGroup:
                 "HttpCode must be like 200 | 200-399 | 200,201 ...",
             )
 
+        if target_type in ("instance", "ip", "alb"):
+            for param in ("protocol", "port", "vpc_id"):
+                if not kwargs.get(param):
+                    param = "VPC ID" if param == "vpc_id" else param.lower()
+                    raise ValidationError(f"A {param} must be specified")
+
+        if vpc_id := kwargs.get("vpc_id"):
```
Comment (bblommers) [inline moto/elbv2/models.py:1096]: We're still supporting Python 3.7, so the `:=` operator will not work unfortunately

Comment (macnev2013) [reply moto/elbv2/models.py]: Updated

File: moto/elbv2/models.py [hunk 2]
```diff
@@ -1080,6 +1091,124 @@ def create_target_group(self, name: str, **kwargs: Any) -> FakeTargetGroup:
...
+            try:
+                self.ec2_backend.get_vpc(kwargs.get("vpc_id"))
+            except InvalidVPCIdError:
+                raise ValidationError(
+                    f"The VPC ID '{kwargs.get('vpc_id')}' is not found"
+                )
+
+        kwargs_patch = {}
+
+        conditions: Dict[str, Any] = {
```
Comment (bblommers) [inline moto/elbv2/models.py:1112]: I like this - makes it very easy to see all defaults in one place

**Review by bblommers — APPROVED** [review]
LGTM - thank you @macnev2013!

**Pull request merged**